### PR TITLE
Added ansible demo playbook

### PR DESCRIPTION
**Pull request by Chris-green-stfc**
ansible demo for installing docker and NGINX for advanced workshop.

### CODE REVIEW

File: AnsiblePlaybooks/ansible-demo/staging
Comment (DavidFair) [inline AnsiblePlaybooks/ansible-demo/staging:1]: We should put some example IPs into this, e.g. `172.16.0.0` to show people how an inventory file looks

File: AnsiblePlaybooks/ansible-demo/ansible-example.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- name: Create a VM
+  hosts: localhost
+  roles:
+  - create_vm
+
+- name: Install Docker
+  hosts: all
+  roles:
+  - install_docker
+  - NGINX
```
Comment (DavidFair) [inline AnsiblePlaybooks/ansible-demo/ansible-example.yaml:19]: Nit: lower case `nginx`

File: AnsiblePlaybooks/ansible-demo/roles/create_vm/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    name: python3-openstacksdk
+    state: present
+  become: true
+
+- name: Boot VM
+  os_server:
+    name: CG_example_VM
+    flavor: l3.nano
+    image: ubuntu-focal-20.04-gui
+    key_name: diz41711
```
Comment (DavidFair) [inline AnsiblePlaybooks/ansible-demo/roles/create_vm/tasks/main.yaml:12]: This vm name and key name needs to be set at the playbook, not role level to make it easy to re-use

**Review by DavidFair — CHANGES_REQUESTED** [review]

**Review by meoflynn — CHANGES_REQUESTED** [review]
Could we add an example inventory to show how to use these playbooks?
The 172.16.0.0 would not work as it is currently

**Comment by DavidFair** [comment]
@Chris-green-stfc is this worth still doing, since we now use dynamic inventories potentially? Either way would be good to get this in before we finish for Winter

**Comment by Chris-green-stfc** [comment]
@DavidFair I updated the staging file, as suggested. I don't think I pinged in the PR channel though

**Pull request closed without merging**